Assembly-CSharp/Memoria/Data/FF9ITEMDto.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Memoria.Prime.CSV;
+
+public class FF9ITEMDto : ICsvEntry
```

Comment:
@FlameHorizon, hi! Let's merge these models.

    If we want to separate DTO and business logic, we need to do this in all code to keep a consistent style.

    The DTO suffix does not make sense in this case, the correct suffix is Csv.

    As far as I understand, this is not just a FF9ITEM, but InitialInventoryEntry or something like this.

I propose in this PR to concentrate on the business task (Changing items given for player at the beginning of new game), and defer refactoring to the future.

Regarding the refactoring itself, when hands reach it, we need to agree on the architecture of the solution. I think that all related entities (descriptions of monsters, items, commands and abilities) should be moved to a separate assembly, and a convenient interface should be provided for obtaining them (for example, in FF PR this is the MasterManager singleton, which allows you to access the in-game database).

At the same time, we can rename the classes so that they correspond to the essence inherent in them.

But do not forget that any such refactoring will also need to be supported in Hades Workshop.

Reply:
Sure. I'll update this PR within day or two :)

Reply:
@FlameHorizon , how's it going? :)